tests/test_other.py

```diff
@@ -7495,6 +7445,7 @@ def test_function_eliminator_replace_variable_value(self):
     self.function_eliminator_test_helper('test-function-eliminator-replace-variable-value.js',
                                          'test-function-eliminator-replace-variable-value-output.js')
 
+  @no_wasm_backend('tests asm.js optimizer')
```

Comment:
How was this working before?

Reply:
It did work, as the test just runs the optimizer directly. But it's redundant (nothing is done differently than the fastcomp path), so I disabled it here.